plexpy/notifiers.py

```diff
@@ -2721,3 +2748,71 @@ def return_config_options(self):
                          ]
 
         return config_option
+
+class HIPCHAT(object):
+
+    def __init__(self):
+        self.apiurl = plexpy.CONFIG.HIPCHAT_URL
+        self.color = plexpy.CONFIG.HIPCHAT_COLOR
+
+    def notify(self, message, subject):
+        if not message or not subject:
+            return
+        
+        text = '(plex) ' + subject.encode('utf-8') + ': ' + message.encode('utf-8')
```

Comment:
I see. Is `(plex)` common across Hipchat? Or do you create your own emoticon? What if someone creates one but doesn't name it `(plex)`?